docs/content/concepts/assets/multi-assets.mdx

```diff
@@ -132,3 +134,23 @@ def my_complex_assets(a, b):
     # d only depends on b
     yield Output(value=b + 1, output_name="d")
 ```
+
+---
+
+## Multi-asset code versions
+
+Like regular assets, multi-assets may assign different code versions for each of their outputs. Versions let you help Dagster track what assets haven't been re-materialized since their code has changed, and avoid performing redundant computation.
```

Comment:
D'you think we should link to the versioning/caching guide here? https://docs.dagster.io/guides/dagster/asset-versioning-and-caching

I feel like I want *something* more about versioning, but not sure if this is the right page - or if we should maybe create something new that's not as complicated as that guide.